src/upsample.jl

```diff
@@ -20,13 +20,16 @@ end
     upsample_bilinear(x::AbstractArray{T,4}, scale::NTuple{2,Real}=(1,1); outsize::Union{Nothing,NTuple{2,Integer}}=nothing)
 
 Upsamples the first 2 dimensions of the array `x` by the upsample factors stored in `scale`,
-using bilinear interpolation.
+using bilinear interpolation. The interpolation is identical to the one used by pytorch.
```

Comment:
let's not mention other DL frameworks in the docs, doesn't look very professional